src/ripple/ledger/tests/View_test.cpp

```diff
@@ -524,6 +525,209 @@ class View_test
     }
 
     void
+    testFlags()
+    {
+        using namespace jtx;
+        Env env(*this);
+
+        auto const alice = Account("alice");
+        auto const bob = Account("bob");
+        auto const carol = Account("carol");
+        auto const gw = Account("gateway");
+        auto const USD = gw["USD"];
+        auto const EUR = gw["EUR"];
+
+        env.fund(XRP(10000), alice, bob, carol, gw);
+        env.trust(USD(100), alice, bob, carol);
+        {
+            // Global freezing.
```

Comment:
One flag I don't see tested is the `NoFreeze` flag. With this flag the issuing account can no longer enable individual freezes, but can still issue a global freeze. It can not disable a global freeze.